docs/learn/learn-proxies.md

```diff
@@ -8,18 +8,30 @@ slug: ../learn-proxies
 ---
 import RPC from "./../../components/RPC-Connection"
 
-Polkadot provides a module that allows users to set proxy accounts to perform a limited number of
-actions on their behalf. Much like the Stash and Controller account relationship in
-[staking](learn-staking.md), proxies allow users to keep one account in cold storage and actively
-participate in the network with the weight of the tokens in that account.
-
-Check out our Polkadot YouTube video that explains what proxy accounts are:
+Much like controller accounts in
+[staking](learn-staking.md), proxies allow users to use an account (it can be a stash account in cold storage or another account in a hot wallet) less frequently but actively
+participate in the network with the weight of the tokens in that account. Proxies can be viewed as an "extreme" version of a controller account, i.e. proxies are allowed to perform a limited amount of actions related to specific [substrate pallets](https://docs.substrate.io/reference/frame-pallets/) on behalf of another account. Check out the video below about proxies.
```

Comment:
```suggestion
participate in the network with the weight of the tokens in that account. Proxies can be viewed as a more powerful and flexible version of a controller account, i.e. proxies are allowed to perform a limited amount of actions related to specific [substrate pallets](https://docs.substrate.io/reference/frame-pallets/) on behalf of another account. Check out the video below about proxies.
```

Reply:
Also please change the last sentence to something like "You can view the video below for more information on proxies and see how they are used" (depending on what exactly is in the video).

In general, we want to avoid extremely casual language like "check out" meaning "see".

Reply:
Thank you Bill, this is very helpful to know as many pages have this kind of informal jargon.